src/java.desktop/macosx/classes/com/apple/laf/AquaKeyBindings.java

```diff
@@ -388,7 +388,9 @@ LateBoundInputMap getTableInputMap() {
             "alt shift TAB", "focusHeader",
             "F8", "focusHeader",
             "ctrl shift UP", "selectFirstRowExtendSelection",
-            "ctrl shift DOWN", "selectLastRowExtendSelection"
+            "ctrl shift DOWN", "selectLastRowExtendSelection",
+            "ctrl shift RIGHT", "selectLastColumnExtendSelection",
+            "ctrl shift LEFT", "selectFirstColumnExtendSelection"
```

Comment:
any specific reason `ctrl shift left / right` is not there for Windows and Linux ?

Reply:
It is there for Windows and Linux too. Its according to the native, in windows it extends to one cell left/right. In Linux there is a miss-match between native `Libre` and swing, might have to change that. I have updated as per Macos native side, where I took `Numbers` as reference.

Reply:
In the test you have mentioned only `ctrl shift up / down` arrow key combination but not the `ctrl shift left / right` for windows and linux.
So, is it missed in test instructions?

Reply:
Yeah, it was missed from previous updates and initial test instructions. Will take it up separately, coz might have to update Linux action command based on native functionality.

Reply:
> Yeah, it was missed from previous updates and initial test instructions. Will take it up separately, coz might have to update Linux action command based on native functionality.

Ok... please check for linux and raise a bug if it is required.